### PR TITLE
Adding CrowdStrike FalconX to generic detonate test playbooks

### PR DESCRIPTION
**Pull request by ohaim1008**
Adding CrowdStrike FalconX to generic detonate file and generic detonate url.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26366

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml [hunk 1]
```diff
@@ -684,13 +685,69 @@ tasks:
...
+        Windows script file: .wsf
+        Javascript: .js
+        Visual Basic: .vbs, .vbe
+        Shockwave Flash: .swf
+        Perl: .pl
+        Powershell: .ps1, .psd1, .psm1
+        Scalable vector graphics: .svg
+        Python: .py
+        Linux ELF executables
+        Email files: MIME RFC 822 .eml, Outlook .msg.      name: Detonate File - CrowdStrike Falcon X
```
Comment (bakatzir) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic.yml:716]:
```suggestion
        Email files: MIME RFC 822 .eml, Outlook .msg.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic.yml [hunk 2]
```diff
@@ -349,13 +350,49 @@ tasks:
...
+    task:
+      id: 07706e7e-800f-408e-8dfa-184841b701f4
+      version: -1
+      name: Detonate URL - CrowdStrike Falcon X
+      playbookName: Detonate URL - CrowdStrike Falcon X
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: |-
+        'Detonate one or more files using the CrowdStrike Falcon Sandbox integration.'
```
Comment (bakatzir) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_URL_-_Generic.yml:366]:
```suggestion
        'Detonate one or more URLs using the CrowdStrike Falcon Sandbox integration.'
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic_CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Added **CrowdStrike Falcon X**.
```
Comment (bakatzir) [inline Packs/CommonPlaybooks/Playbooks/playbook-Detonate_File_-_Generic_CHANGELOG.md:2]: CHANGELOG is not needed anymore, no need to make changes in this file.

**Comment by amshamah419** [comment]
Looks good

**Pull request merged**